reporter/grpc/language-agent/Tracing.proto

```diff
@@ -19,60 +19,79 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "org.apache.skywalking.apm.network.language.agent.v2";
+option java_package = "org.apache.skywalking.apm.network.language.agent.v3";
 option csharp_namespace = "SkyWalking.NetworkProtocol";
-option go_package = "github.com/SkyAPM/go2sky/reporter/grpc/language-agent-v2";
+option go_package = "github.com/SkyAPM/go2sky/reporter/grpc/language-agent";
```

Comment:
Do we still need to copy all these, as we already have repo https://github.com/SkyAPM/skywalking-data-collect-protocol-for-go?

Reply:
I want to keep the copy.

Reply:
Sure, we could keep this for now. But we need to consider it in the future.